src/logic/tokens/utils/formatAmount.ts

```diff
@@ -47,3 +49,6 @@ export const formatAmountInUsFormat = (amount: string): string => {
   const numberFloat: number = parseFloat(amount)
   return usNumberFormatter.format(numberFloat).replace('$', '')
 }
+
+export const humanReadableTokenAmount = (amount: string, decimals: number): string =>
```

Comment:
I think this function should be defined in `src/utils`. It can be used for other things than a token value. 
Also the name should be more generic.